tests/fn_boundary.json

```diff
@@ -38,15 +38,638 @@
+      "view": {
+        "resource": "Observation",
+        "status": "active",
+        "select": [
+          {
+            "column": [
+              {
+                "name": "id",
+                "path": "id",
+                "type": "id"
+              },
+              {
+                "name": "decimal",
+                "path": "value.ofType(Quantity).value.highBoundary()",
+                "type": "decimal"
+              }
+            ]
+          }
+        ]
+      },
+      "expect": [
+        {
+          "id": "o1",
+          "decimal": 1.09999999
+        },
+        {
+          "id": "o2",
+          "decimal": null
+        },
+        {
+          "id": "o3",
+          "decimal": null
+        },
+        {
+          "id": "o4",
+          "decimal": null
+        },
+        {
+          "id": "o5",
+          "decimal": null
+        },
+        {
+          "id": "o6",
+          "decimal": null
+        },
+        {
+          "id": "o7",
+          "decimal": null
+        },
+        {
+          "id": "o8",
+          "decimal": null
+        },
+        {
+          "id": "o9",
+          "decimal": null
+        },
+        {
+          "id": "o10",
+          "decimal": null
+        },
+        {
+          "id": "o11",
+          "decimal": null
+        },
+        {
+          "id": "o12",
+          "decimal": null
+        },
+        {
+          "id": "o13",
+          "decimal": null
+        },
+        {
+          "id": "o14",
+          "decimal": null
+        },
+        {
+          "id": "o15",
+          "decimal": null
+        },
+        {
+          "id": "o16",
+          "decimal": null
+        },
+        {
+          "id": "o17",
+          "decimal": null
+        },
+        {
+          "id": "o18",
+          "decimal": null
+        },
+        {
+          "id": "o19",
+          "decimal": null
+        },
+        {
+          "id": "o20",
+          "decimal": null
+        },
+        {
+          "id": "o21",
+          "decimal": null
+        },
+        {
+          "id": "o22",
+          "decimal": null
+        },
+        {
+          "id": "o23",
+          "decimal": null
+        },
+        {
+          "id": "o24",
+          "decimal": null
+        },
+        {
+          "id": "o25",
+          "decimal": null
+        },
+        {
+          "id": "o26",
+          "decimal": 1.58799999
+        },
+        {
+          "id": "o27",
+          "decimal": 10235.66999999
+        },
+        {
+          "id": "o28",
+          "decimal": 353600.89123999
+        },
+        {
+          "id": "o29",
+          "decimal": -353600.89123999
+        },
+        {
+          "id": "o30",
+          "decimal": 0.19999999
+        },
+        {
+          "id": "o31",
+          "decimal": null
+        },
+        {
+          "id": "o32",
+          "decimal": 0.00799999
+        },
+        {
+          "id": "o33",
+          "decimal": 15.0809999
```

Comment:
This test failed, because the input value was 15.080. However, JS removes 0 in the end of value, so the input becomes 15.08, and the returned value is 15.08999999, but it should be 15.0809999.